src/pages/postEventSurvey/postEventSurvey.tsx

```diff
@@ -107,34 +148,60 @@ function ExitSurveyModal() {
                   </IonSelect>
               </IonItem>
 
+              <IonItem className="modal-field">
+                <IonLabel position="floating">What app did you use for Walktober?</IonLabel>
```

Comment:
@bailee-crandley is this fine? or do you want to change the wording?

Reply:
That's fine for now. I have to go in and change the wording for a few of these questions but I was just going to wait until after this PR is merged. I would change it to "If you used a health app to load your steps data, which one did you use?" but I can make that change when I do the other changes as well.